src/utils.js

```diff
@@ -246,6 +247,24 @@ function buildStalenessResponseObject(isStale, timestamp) {
     };
 }
 
+function getLimitFromAlarmRange(alarmRange) {
+    let limits = {};
+    alarmRange.forEach(alarm => {
+        limits[alarm.level] = {
+            low: {
+                color: limitConfig[alarm.level],
+                value: alarm.minInclusive || alarm.minExclusive
```

Comment:
I know this was like this before, but I think using `alarm.minInclusive ?? alam.minExclusive` is safer? I'm not sure though, this way if minInclusive is `0` then it would actually use it? If we're just checking for undefined, then I think `??` is the way to go.

Reply:
You're right. `alarm.minInclusive ?? alam.minExclusive` is what we should go with since we do want to allow 0. `||` would not allow a 0.